README.md

```diff
@@ -21,11 +21,11 @@ cd ios && pod install
 
 ### 4. Start the app!
 
-`npm start` or `npx react-native start` if you have not installed `react-native-cli`
+`npm start` or `npx react-native start` (if you don't have `react-native-cli` installed)
 
 ## How it works
 
-Integrations in React Native must inform the `integrationContext` parameter with the value `InAppBrowserNotify` in the establish data, as in the example below:
+Integrations in React Native must inform the `integrationContext` parameter with the value `InAppBrowserNotify` in the `establishData`, as in the example below:
```

Comment:
I think this is a bit better:

For React Native integrations, set the `integrationContext` parameter to `InAppBrowserNotify` within the establishData function, as shown in the example below: